src/audio_analyze.py

```diff
@@ -375,6 +373,7 @@ def main(argv):
         debug=options.debug,
         min_separation_msec=options.min_separation_msec,
         min_correlation_factor=options.correlation_factor,
+        echo_analysis=options.echo_analysis,
```

Comment:
I assume `echo_analysis` is the detection of 2x consecutive audio signals (for videoconf audio latency measurements). Is this described somewhere?

Reply:
A sentence in the help is all there is at the moment. The read me could be updated with this though.